startvs.cmd

```diff
@@ -14,10 +14,11 @@ SET PATH=%DOTNET_ROOT%;%PATH%
 SET sln=%1
 
 IF NOT EXIST %DOTNET_ROOT%\dotnet.exe (
-    echo .NET Core has not yet been installed. Run `build.cmd -restore` to install tools
-    exit /b 1
+    restore.cmd
 )
 
+echo ProTip^: You can drag and drop a solution file into startvs.cmd
```

Comment:
Ooh, that's cool.

Reply:
lol, I was surprised this was a surprise to my team, so I thought I'd evangelize it a bit more.